src/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -8,7 +8,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.0</NuGetTargetMoniker>
```

Comment:
Add a comment above here that says we want to continue to target 1.0 so that this will be work on WP8.